open-apparel-registry/open-apparel-registry
# Prepare OAR code for read-only database

## Overview

This PR updates the UI and some of the django API in preparation for making the database read only.

Connects #2160 

## Demo

![Screen Shot 2022-09-28 at 11 39 58 AM](https://user-images.githubusercontent.com/8356789/192823738-fd48720b-44d9-430b-ad13-0206f8bfc119.png)


## Notes

The facility download logic was embedded in the `logDownload` reducer so it and related actions were renamed to `facilityDownload`.

There might be additional components/tests/views that I missed that can be deleted, but the UI should not utilize these features.

## Testing Instructions

* Go to homepage
* Contribute/Register button should be greyed out and unclickable.
* Log in
* Contribute button should be greyed out and unclickable
* Go to a facility page
* The facility CTA's at the bottom of the side bar should be removed
* Go to an unclaimed facility page
* The "Claim facility" header should not be present
* Attempt to go to user settings
* A "Not available" message should be displayed instead
* Go to My Facilities
* Facility claim details should not be viewable

## Checklist
